webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/KnoraGroupService.scala

```diff
@@ -7,12 +7,22 @@ package org.knora.webapi.slice.admin.domain.service
 
 import zio.Chunk
 import zio.Task
+import zio.ZIO
 import zio.ZLayer
 
+import dsp.errors.DuplicateValueException
+import org.knora.webapi.responders.IriService
+import org.knora.webapi.slice.admin.api.GroupsRequests.GroupCreateRequest
 import org.knora.webapi.slice.admin.domain.model.GroupIri
+import org.knora.webapi.slice.admin.domain.model.GroupName
 import org.knora.webapi.slice.admin.domain.model.KnoraGroup
+import org.knora.webapi.slice.admin.domain.model.KnoraProject
 
-case class KnoraGroupService(knoraGroupRepo: KnoraGroupRepo) {
+case class KnoraGroupService(
+  knoraGroupRepo: KnoraGroupRepo,
+  knoraProjectRepo: KnoraProjectRepo,
```

Comment:
**suggestion**: You can remove this unused dependency

Reply:
Right, forgot to remove it during the cleanup. Will remove it in follow up PR.

Reply:
Since you merged it without my suggestion and did not leave a comment as to wether this will be addressed in a later PR:
https://github.com/dasch-swiss/dsp-api/pull/3192